plugins/example-todo-list-backend/src/plugin.ts

```diff
@@ -26,7 +26,7 @@ import { createRouter } from './service/router';
  * @public
  */
 export const exampleTodoListPlugin = createBackendPlugin({
-  pluginId: 'exampleTodoList',
+  pluginId: 'todolist',
```

Comment:
This id should match the same id used [here](https://github.com/backstage/backstage/blob/5fc874e236aeefbbdb92f701b647a1543aef50ec/plugins/example-todo-list/src/components/TodoListPage/TodoListPage.tsx#L79-L80) otherwise we get a 404 error in the fetch api requests